src/Admin/MessageAdmin.php

```diff
@@ -95,7 +95,7 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
 
         $datagridMapper
             ->add('type')
-            ->add('state', null, [], ChoiceFilter::class, ['choices' => $class::getStateList()])
+            ->add('state', null, [], 'choice', ['choices' => $class::getStateList()])
```

Comment:
This doesn't look like the right fix, it should really work.

Reply:
That’s because the second param is for the filter type, and the fourth is for the form type, so I think this is a wrong fix. The correct way should be to load ChoiceType from symfony namespace instead of ChoiceFilter. Can you check this @pcdummy

Reply:
@jordisala1991 Did so, thanks.

Reply:
Did you forget to force push your changes?

Reply:
Forgot to add the file, yes.